docs/api.swagger.json

```diff
@@ -3135,12 +3135,12 @@
       "authorizationUrl": "/oauth2/auth",
       "tokenUrl": "/oauth2/token",
       "scopes": {
-        "offline": "A scope required when requesting refresh tokens (alias for `offline`)",
+        "offline": "A scope required when requesting refresh tokens (alias for `offline_access`)",
         "offline_access": "A scope required when requesting refresh tokens",
```

Comment:
Thank you! This file is autogenerated. The appropriate line to fix is here: https://github.com/ory/hydra/blob/master/doc.go#L44

Reply:
Oh boy, not again 😁 This time I thought I had it right. Thanks for the hint :v: